contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.0.0--2.1.0.sql

```diff
@@ -3,6 +3,31 @@
  
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
 
+-- Drops a view if it does not have any dependent objects.
+-- Is a temporary procedure for use by the upgrade script. Will be dropped at the end of the upgrade.
+-- Please have this be one of the first statements executed in this upgrade script. 
+CREATE OR REPLACE PROCEDURE babelfish_drop_deprecated_view(schema_name varchar, view_name varchar) AS
+$$
+DECLARE
+    error_msg text;
+    query1 text;
```

Comment:
FWIW there's really no need for `query1` or `query2`; we could simply do `EXECUTE format($$....$$, schema_name, view_name)`.

Reply:
Ack. Good to know.